stockflux-securities-master/src/components/Alert.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Alert.css";
+
+const Alert = ({ messages, type, className }) => {
```

Comment:
Remember, we said this should be able to accept a string or an array so you should first check if it is an array and if it isn't then convert it to one...

    messages = Array.isArray(messages) ? messages : [messages];

or you could just as easily do it when you call `.map(...)`

    {(Array.isArray(messages) ? messages : [messages]).map((message, index) => {

Reply:
My mistake, I thought in your last review you told me to remove this check. I'll add it back in but without the useEffect and state

Reply:
done now

Reply:
To clarify, my last review said that you don't need state since the argument (or prop) is passed.